setup/github-bot/ansible-playbook.yaml

```diff
@@ -0,0 +1,73 @@
+---
+- hosts: github-bot
+  remote_user: root
+  gather_facts: False
+
+  tasks:
+
+    - name: Bootstrap | Install baseline packages
+      raw: apt-get install -y aptitude python-apt apt-transport-https
```

Comment:
no no thats perfect but if I'm hallucinating and its not needed then we can just skip it -- I'm just sure I encountered a box that didn't have it installed. How about we stick to facts and you just install it as you chose to. Sorry for the noise :/